package.json

```diff
@@ -89,6 +88,7 @@
     "ejs": "^2.6.1",
     "express": "^4.16.3",
     "express-static-gzip": "^1.1.3",
+    "masq-lib": "git+https://github.com/QwantResearch/masq-lib.git#maps-integration",
```

Comment:
I think this dependency is only required on build, and should be moved to "devDependencies".  

This currently blocks the docker image build, as it also depends on other devDependencies.

Reply:
You're righ ! I moved masq-lib to devDependencies